model/task/task.go

```diff
@@ -273,6 +271,16 @@ type Task struct {
 	IsEssentialToSucceed bool `bson:"is_essential_to_succeed" json:"is_essential_to_succeed"`
 }
 
+// StepbackInfo is used to indicate what task to stepback to.
```

Comment:
this isn't quite right, is it? It stores information about stepback for the task more generally. (I imagine this comment as is would be the definition of NextStepbackTaskId)